inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/gather_tree.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "single_layer_tests/gather_tree.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+        //InferenceEngine::Precision::FP32,
```

Comment:
Why `FP32` is commented?

Reply:
Because according to the specification FP32 input tensors have to be filled with integer indexes. But I couldn't find a way to initialize FP32 parameters with integer numbers.

Reply:
FP32 precision https://github.com/openvinotoolkit/openvino/pull/2006/commits/dcc5579d9c61ce1de7051cbbc9cc9d412f675123